schemas/Manual.items.schema.json

```diff
@@ -26,76 +26,78 @@
                     "description": "The unique name of the item. Do not use () or : in the name",
                     "type": "string"
                 },
-                "progression": {
-                    "description": "(Optional) Is this item needed to unlock locations? Defaults to false if not included.",
-                    "default": true,
-                    "type": "boolean"
-                },
-                "count": {
-                    "description": "(Optional) Number of copy of this item in the pool.",
-                    "type": "integer",
-                    "default": 1
-                },
                 "category": {
                     "description": "(Optional) The category(ies) this item fit in",
                     "type": "array",
                     "items": {
-                        "$ref": "#/definitions/Category"
+                        "type": "string"
                     },
                     "minItems": 1,
                     "uniqueItems": true
                 },
-                "useful": {
-                    "description": "(Optional) Is this item usefull to have? Used for items that are not progression but you still want the rando to really use.",
-                    "type": "boolean"
+                "count": {
+                    "description": "(Optional) Number of copy of this item in the pool.",
+                    "type": "integer",
+                    "default": 1
+                },
+                "value": {
+                    "description": "(Optional) A dictionary of values this item has in the format {\"name\": int,\"otherName\": int} \nUsed with the {ItemValue(Name: int)} in location requires \neg. \"value\": {\"coins\":10} mean this item is worth 10 coins",
+                    "type": "object",
+                    "patternProperties": {
+                        "^.+$": {
+                            "anyOf": [
+                                {
+                                    "type": "integer",
+                                    "description": "A value that this item has must be 'name':integer \neg. \"coins\": 10"
+                                }
+                            ]
+                        }
+                    }
+                },
+                "progression": {
+                    "description": "(Optional) Is this item needed to unlock locations? Defaults to false if not included.",
```

Comment:
Do we need to keep the default in the description block if we fixed how the `default` tag works?

Reply:
Good catch! Removed those 😄